eoscostarica/guide.eoscostarica.io
# 445 update boilerplate readme

### Title

Updated boilerplate and deleted sections that are already visible in GitHub and changed infrastructure diagram.

- resolves eoscostarica/full-stack-boilerplate#39 

### Steps to test
1. Open the Readme and see changes to grammar, sections and the new infrastructure diagram.


#### CheckList
